Magento_MultipleWishlist/web/js/multiple-wishlist-mixin.js

```diff
@@ -98,6 +98,12 @@ define([
             var element = $(e);
             var buttonName = element.text().trim();
             var generalParams = element.data('multiplewishlist');
+            /**
+             * temp solution for Recently Viewed multiple wishlist console error (#86220)
```

Comment:
can you add `TODO` in the comment? so that this doesn't get lost and stay like this forever